qemu/tests/cfg/blockdev_mirror_qemuio_ready_job.cfg

```diff
@@ -0,0 +1,26 @@
+- blockdev_mirror_qemuio_ready_job:
+    only Linux
+    start_vm = no
+    qemu_force_use_drive_expression = no
+    type = blockdev_mirror_qemuio_ready_job
+    virt_test_type = qemu
+    images += " data1"
+    source_images = data1
+    target_images = mirror1
+    remove_image_data1 = yes
+    force_create_image_data1 = yes
+    backup_options_data1 = sync
+    sync = full
+    storage_pools = default
+    storage_pool = default
+
+    image_size_data1 = 2G
+    image_size_mirror1 = ${image_size_data1}
+    image_format_data1 = qcow2
```

Comment:
Maybe we don't need to limit the data disk type since the original product bug of the test case was easily to occur on raw format.

Reply:
Talked to Aihua, she proved this case is not must related to raw format, so will use qcow2 format as we always do. Will send new patch to support raw format in the future.